index.html

```diff
@@ -66,7 +66,140 @@ <h5>Pay as You Go</h5>
+                    required
+                 />
+                </div>
+                <div class="what">
+                  <label for="last-name">Last name *</label>
+                  <input 
+                    type="text" 
+                    id="last-name"
+                    required
+                  />
+                </div>
+            </div>
+
+            <div class="address">
+                <div class="what">
+                  <label for="address1">Address *</label>
+                  <input 
+                    type="text"
+                    id="address1"
+                    required
+                  />
+                </div>
+            </div>
+
+            <div class="address">
+                <div class="what">
+                  <label for="address2">Address 2</label>
+                  <input 
+                    type="text"
+                    id="address2"
+                  />
+                </div>
+            </div>
+
+            <div class="fieldset">
+                <div class="what">
+                  <label for="city">City *</label>
+                  <select name="city" id="city" required>
+                    <option value="" disabled selected>Select your city...</option>
+                    <option value="london">London</option>
+                    <option value="birmingham">Birmingham</option>
+                    <option value="manchester">Manchester</option>
+                    <option value="bristol">Bristol</option>
+                    <option value="liverpool">Liverpool</option>
+                  </select>
+                </div>
+                <div class="what">
+                  <label for="postcode">Postcode *</label>
+                  <input 
+                    type="text"
+                    id="postcode"
+                    required
+                  />
+                </div>
+            </div>
+
+            <div class="radio-box">
+              <legend>Select a color</legend>
+              <ul>
+                <li>
+                    <input type="radio" id="karma-orange" name="color" value="karma-orange" checked/>
+                    <label for="karma-orange">Karma Orange</label>
+                </li>
+                <li>
+                  <input type="radio" id="space-grey" name="color" value="space-grey"/>
+                  <label for="karma-orange">Space Grey</label>
+              </li>
+              </ul>
+            </div>
+
+            <div class="check-box">
+              <input type="checkbox" id="termsandconditions" required>
+              <label for="termsandconditions">By placing your order you agree to <a href="">Karma's Terms and Conditions</a> *</label>
+            </div>
+
+            <div>
+                <button>Place my order</button>
+            </div>
+        </form>
+    </section>
+
+    <section class="section2-img">
+        <img src="level-2/store-image_by-andrew-neel-unsplash.jpg" alt="">
```

Comment:
Maybe this is a decorative image, like the one in the new section on index.html, so you could use CSS `background-image` to add it